Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8
+
+RUN apt-get update && \
+    apt-get -y install gcc build-essential
+
+RUN mkdir -p /usr/src/app/heksher
+
+WORKDIR /usr/src/app/heksher
+
+RUN pip install poetry
+
+RUN poetry config virtualenvs.create false
+COPY pyproject.toml poetry.lock* ./
+RUN poetry install --no-dev --no-root -E alembic
+# needed for poetry version to work
+RUN pip install requests
```

Comment:
install poetry not via pip and it will have it's own venv and will save you patches.